lib/facade_converter.ts

```diff
@@ -9,9 +9,8 @@ import {MergedType} from './merge';
 const FACADE_DEBUG = false;
 const FACADE_NODE_MODULES_PREFIX = /^(\.\.\/)*node_modules\//;
 
-// These constants must be kept in sync with package:func/func.dart which
-// provides a cannonical set of typedefs defining commonly used function types
-// to simplify specifying function types in Dart.
+// These constants must be kept in sync with package:func/func.dart which provides a canonical set
```

Comment:
Add a TODO to remove the dependency on package:func/func.dart
Dart now has good Function type syntax that we can use in this case!
E.g.

`int Function([String x])`
defines a function type with an optional String parameter.